node/src/main/scala/coop/rchain/node/configuration/Configuration.scala

```diff
@@ -102,9 +102,13 @@ object Configuration {
           s"The value 'pos-multi-sig-quorum' should be less or equal the length of 'pos-multi-sig-public-keys' " +
           s"(the actual values are '$posMultiSigQuorum' and '$posMultiSigPublicKeysLength' respectively)"
       )
-      System.exit(1)
     }
 
+    /* if (nodeConf.casper.genesisBlockData.posVaultPubKey.isEmpty) {
```

Comment:
why this is commented out?

Reply:
It's temporally. I wanted to understand why integration tests fail